### ISSUE TITLE
sfpowerkit:pool:create not retrieving the alias correctly anymore

### ISSUE THREAD

**Issue by jverelst** [dxatscale/sfpowerkit#726]
Hello,

We just upgraded our CICD pipeline Docker image, to include the latest versions of SFDX and the sfpowerkit plugin. I noticed that the combination of these two now result in the following output:
```
Relaxing ip ranges for scratchOrg with user test-pkhpxazjwixe@example.com
Polling for Retrieval Status
Ip range to set : 0.0.0.0-255.255.255.255
Polling for Deployment Status
Ip range is successfully set in test-pkhpxazjwixe@example.com
Executing command: bash scripts/scratch-pool-full-after-create.sh  test-pkhpxazjwixe@example.com  salesforcedx@renewi.com 
Executing script for undefined with username: test-pkhpxazjwixe@example.com
Script Execution result is being written to script_exec_outputs/undefined.log, Please note this will take a significant time depending on the  script being executed
```

Note the 'undefined' twice: this is the alias of the Scratch org that is being retrieved from the json output of the `sfdx force:org:create` command:

https://github.com/dxatscale/sfpowerkit/blob/f0f7ee544c13fb1de95187b8aabc66f928d71547/src/utils/scratchOrgUtils.ts#L103

The problem is that the output for `sfdx force:org:create` doesn't return the alias anymore since 7-jan-2022, due to this commit:
https://github.com/forcedotcom/sfdx-core/commit/109e9c3f8b21fa612303140338105bb4144e2e97

The problem we are now facing is that if we create a pool of 10 scratchorgs, there is only 1 output file `undefined.log` which gets overwritten by 10 threads simultaneously. We used to have `SO1.log`, `SO2.log`, etc.

Could this be updated to just use the alias as it is being calculated in lines 91..95?

**Comment by jverelst** [comment]
Addition: I think this was introduced by version 7.164.2 of SFDX: https://github.com/forcedotcom/cli/blob/main/releasenotes/sfdx/README.md#71642-aug-18-2022

It says that 'force:org:beta:create' has been promoted to 'force:org:create', which probably introduced this behavior

**Comment by Caitlyn-Mills** [comment]
Thanks for raising! We will look into using force:org:legacy:create or supplying a generic alias for each SO until we can identify a proper fix. 
![image](https://user-images.githubusercontent.com/63215232/187112048-4f13c588-be79-47ad-849b-beefa463eab7.png)

I would also recommend looking into whether you can switch over to creating your pool through the sfpowerscripts:orchestrator:prepare command https://github.com/dxatscale/sfpowerscripts

**Comment by Caitlyn-Mills** [comment]
@jverelst This is fixed in 5.0.0 please reopen if you still experience the issue.
